Dockerfile

```diff
@@ -3,8 +3,8 @@
 # TO_BUILD_AND_RUN: docker-compose up
 # NOTES:            Currently this is one big dockerfile and non-optimal.
 
-
-ARG OPENSTUDIO_VERSION=2.8.0
+#may include suffix
+ARG OPENSTUDIO_VERSION
```

Comment:
looks like this is now going to be a required input... that is if the user does not specify the input then this won't build.